ui/js/values.js

```diff
@@ -108,6 +123,8 @@ treeherder.value("thPlatformMap", {
     "horizon-win64": "Horizon Windows x64",
 
     "gecko-decision": "Gecko Decision Task",
+    "lint": "Linting",
+    "release-mozilla-release-": "Balrog Publishing",
```

Comment:
Either that or maybe the patch in bug 1387685 needs to be more detailed to properly parse that buildername, and I can possibly drop that line from here.